test/config.yaml

```diff
@@ -0,0 +1,86 @@
+#### General Settings ####
+# location of system level settings
+settings:
+  title: My Very Cool Project
+  Author: Bob
+
+  # path like settings
+  data: /data/bob/original_data
+
+  # conda environment names
+  python2: py2.7
+
+  # names to access specific envs by
+  env: HOME
+
+#### Experiment Level Settings ####
+# experiment level settings, settings that apply to all samples
+exp.settings:
+
+  # Sample information relating sample specific settings to sample ids
+  sampleinfo: sample_metadata.csv
+
+  # it would be nice to be able define a setting here that applies to all
+  # samples, or define for each sample in the sampleinfo table case they are
+  # different.
+  fastq_suffix: '.fastq.gz'
+  # Need to some way to specify annotation to use, maybe here is not the best
+  # place.
+  annotation:
```

Comment:
Yeah, the iterative junctions are going to get tricky. I have some ideas about that but it needs more testing.

Currently swapping flybase versions should be as easy as swapping annotation filenames. I think the bigger question is whether or not we should support multiple "flavors" of annotations in a single workflow. For now I'm leaning towards just supporting a single one to keep things simple, and having downstream rules concatenate de novo junctions onto that single single annotation.